components/Itemlist.js

```diff
@@ -30,6 +32,12 @@ const StyledItemListContainer = styled.section`
   margin-top: 10px;
   padding: 2px;
 
+  .fav {
+    width: 2rem;
+    position: absolute;
+    right: 15px;
+  }
+
```

Comment:
I think you dont have to work with a className because you could use FavoriteIcon as a styled Component itself, like: 
`const StyledFavoriteIcon = styled(FavoriteIcon)`

but I guess your version is fine, too :)

Reply:
ok nice to know i will change it with the next US :)